packages/features/users/components/UserTable/BulkActions/DeleteBulkUsers.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Table } from "@tanstack/react-table";
+import { BanIcon } from "lucide-react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { Dialog, DialogTrigger, ConfirmationDialogContent, Button, showToast } from "@calcom/ui";
+
+import type { User } from "../UserListTable";
+
+interface Props {
+  table: Table<User>;
+}
+
+export function DeleteBulkUsers({ table }: Props) {
+  const { t } = useLocale();
+  const selectedRows = table.getSelectedRowModel().flatRows.map((row) => row.original); // Get selected rows from table
```

Comment:
Get selected row values from react-table

Reply:
nit: This component needs just the users. Maybe we can pass just that prop to DeleteBulkUsers
<img width="464" alt="Screenshot 2023-08-02 at 5 15 39 PM" src="https://github.com/calcom/cal.com/assets/1780212/9b034349-6920-495f-9934-2710f4827ad5">